.github/workflows/sourcegraph.yml

```
@@ -19,8 +19,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: coursier/setup-action@v1.2.0-M2
-      - run: cs install --contrib lsif-java
-      - run: lsif-java index
+      - run: cs install --contrib scip-java
+      - run: scip-java index
```

```suggestion
      - run: cs launch --contrib scip-java -- index
```

Not sure why, but I could run `scip-java index` by `cs launch` instead of `cs install` 🤔

---

Yea I noticed that as well before. I suppose it's maybe just a good idea to do that now so the build stops being red, and then I'll change it later when the install works correctly.